src/export/digitalPlanning/model.ts

```diff
@@ -411,12 +423,31 @@ export class DigitalPlanning {
             number: this.passport.data?.["property.EPC.number"],
           }),
         },
+        parking: this.getExistingParking(),
       } as LondonProperty;
     } else {
       return baseProperty as UKProperty;
     }
   }
 
+  private getExistingParking(): LondonProperty["parking"] {
+    const parking = {};
+    PARKING_TYPES.forEach((type) => {
+      if (type === "offStreet.residential") {
+        parking["offStreet"] = {
+          residential: {
+            count: this.passport.data?.[`property.parking.${type}`] || 0,
+          },
+        };
```

Comment:
What's the clever syntax I'm missing here to be able to pass a dot-separated string to create a new nested object property?

Reply:
I think what you're looking for is `set()` from Lodash ([docs](https://lodash.com/docs/4.17.15#set))

Example in out code here - https://github.com/theopensystemslab/planx-core/blob/main/src/export/digitalPlanning/model.ts#L600-L604

Reply:
Yep updated throughout now - was just being avoidant to re-writing this section of mappings using `set()` in case there was another route - but this is definitely cleanest :+1: